artifactory/commands/utils/result.go

```diff
@@ -31,3 +39,50 @@ func (r *Result) SetFailCount(failCount int) {
 func (r *Result) SetReader(reader *content.ContentReader) {
 	r.reader = reader
 }
+
+func UnmarshalDeployableArtifacts(filePath string) (*Result, error) {
```

Comment:
Let's add a description for this function, helping the random reader understand its functionality.